main.py

```diff
@@ -86,55 +88,61 @@ async def keep_alive_task():
     except Exception as e:
         logger.exception("An error occurred while trying to perform keep alive task:", exc_info=e)
 
-
-
 @bot.event
-async def on_message(message):
+async def on_message(message:Message):
     # Registra en el log cuando se recibe un mensaje
     logger.debug('Received a message.')
-    
+
     # Ignora los mensajes del propio bot o de canales que no sean el canal de origen
     if message.author == bot.user or message.channel.id != id_canal_origen:
         logger.debug('Message is from the bot itself or not from the origin channel, ignoring.')
         return
 
     try:
-        # Crea un embed para enviar al canal
-        embed = discord.Embed(color=discord.Color.blue())
-        bloques = re.split(r'(\[tituloinicio\].*?\[titulofin\])', message.content, flags=re.DOTALL)
-        eventos_para_botones = []
-
-        for bloque in bloques:
-            if '[tituloinicio]' in bloque and '[titulofin]' in bloque:
-                titulo = re.search(r'\[tituloinicio\](.*?)\[titulofin\]', bloque).group(1)
-                embed.add_field(name=titulo, value='', inline=False)
-            else:
-                eventos = re.split(r'(\[evento\d+\])', bloque, flags=re.DOTALL)
-                for i in range(1, len(eventos), 2):
-                    evento_texto = eventos[i + 1].strip()
-                    evento_texto = re.sub(r'(\[evento\d+\])\s+(\S+)', r'\1 **\2**', evento_texto)
-                    
-                    if '[remember]' in evento_texto:
-                        eventos_para_botones.append(evento_texto)
-                        evento_texto = evento_texto.replace('[remember]', '').strip()
-                    
-                    evento_texto = re.sub(r'\[linki\](https?://[^\s\[]+)\[linke\](\S+)', r'[\2](\1)', evento_texto)
-                    evento_texto = evento_texto.replace(')(', ') - (')
-                    embed.add_field(name='', value=evento_texto, inline=False)
-
-        # Crea una vista con botones si es necesario
-        view = ReminderView(eventos_para_botones) if eventos_para_botones else None
-
-        # Envía el mensaje al canal de destino
+        # Preparar y enviar el mensaje al canal de destino
         for id_canal in ids_canales_destino:
```

Comment:
As you can see, I will use the id_canal as the target channel ID for the analysis button.